prov/efa/src/rdm/rxr.h

```diff
@@ -79,6 +79,11 @@ static inline void rxr_poison_mem_region(void *ptr, size_t size)
 }
 #endif
 
+/*
+ * The alignment to support LL128 protocol in NCCL
+ */
+#define RXR_128_BYTES_ALIGNMENT (128)
```

Comment:
we are moving away from `RXR` to `EFA_RDM`. Also the naming is confusing. what does "128_BYTES_ALIGNMENT" mean?

Reply:
I removed this macro in latest revision.